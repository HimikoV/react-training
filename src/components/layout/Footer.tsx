import { Box, Typography, Grid } from "@mui/material";
const Footer = () => {
  return (
    <Grid
      sx={{
        px: 10,
        py: 4,
        bgcolor: "#3d405b",
        justifyContent: "space-around",
        alignItems: "center",
        left: 0,
        bottom: 0,
        flexDirection: "row",
      }}
    >
      <Box>
        <Box sx={{ borderBottom: "1px solid #f4f1de" }}>
          <Typography variant="caption" color="secondary">
            FAQ
          </Typography>
        </Box>
        <Box>
          <Typography variant="caption" color="secondary">
            hello world
          </Typography>
        </Box>
        <Box>
          <Typography variant="caption" color="secondary">
            hello world
          </Typography>
        </Box>
      </Box>
      <Box>
        <Box sx={{ borderBottom: "1px solid #f4f1de" }}>
          <Typography variant="caption" color="secondary">
            Contact
          </Typography>
        </Box>
        <Box>
          <Typography variant="caption" color="secondary">
            hello world
          </Typography>
        </Box>
        <Box>
          <Typography variant="caption" color="secondary">
            hello world
          </Typography>
        </Box>
      </Box>
      <Box>
        <Box sx={{ borderBottom: "1px solid #f4f1de" }}>
          <Typography variant="caption" color="secondary">
            About
          </Typography>
        </Box>
        <Box>
          <Typography variant="caption" color="secondary">
            hello world
          </Typography>
        </Box>
        <Box>
          <Typography variant="caption" color="secondary">
            hello world
          </Typography>
        </Box>
      </Box>
    </Grid>
  );
};

export default Footer;
