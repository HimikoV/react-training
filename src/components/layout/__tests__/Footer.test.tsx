import { describe, it } from "@jest/globals";
import renderer from "react-test-renderer";
import Footer from "../Footer";

describe("testing <Footer/> render", () => {
  it("Footer Renders Correctly", () => {
    const footer = renderer.create(<Footer />).toJSON();
    expect(footer).toMatchSnapshot();
  });
});
