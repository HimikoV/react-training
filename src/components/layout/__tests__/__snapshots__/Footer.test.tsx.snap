// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing <Footer/> render Footer Renders Correctly 1`] = `
<div
  className="MuiGrid-root css-1b3rmzp-MuiGrid-root"
>
  <div
    className="MuiBox-root css-0"
  >
    <div
      className="MuiBox-root css-180jf3a"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        FAQ
      </span>
    </div>
    <div
      className="MuiBox-root css-0"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        hello world
      </span>
    </div>
    <div
      className="MuiBox-root css-0"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        hello world
      </span>
    </div>
  </div>
  <div
    className="MuiBox-root css-0"
  >
    <div
      className="MuiBox-root css-180jf3a"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        Contact
      </span>
    </div>
    <div
      className="MuiBox-root css-0"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        hello world
      </span>
    </div>
    <div
      className="MuiBox-root css-0"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        hello world
      </span>
    </div>
  </div>
  <div
    className="MuiBox-root css-0"
  >
    <div
      className="MuiBox-root css-180jf3a"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        About
      </span>
    </div>
    <div
      className="MuiBox-root css-0"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        hello world
      </span>
    </div>
    <div
      className="MuiBox-root css-0"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-1061lhe-MuiTypography-root"
      >
        hello world
      </span>
    </div>
  </div>
</div>
`;
